src/redux/actions/posts.js

```diff
@@ -0,0 +1,4 @@
+import { ADD_POSTS_POST, DELETE_POSTS_POST } from '../types/posts';
+
+export const addPost = (text) => ({type: ADD_POSTS_POST, payload: text});
+export const deletePost = (id) => ({type: DELETE_POSTS_POST, payload: id});
```

Comment:
Can we just name it like ADD_POST, DELETE_POST? =)

Reply:
POSTS is the name of the reducer.
ADD_POST, DELETE_POST are too common names and can be used for other reducers.
We need to name the action type more unique way. 
I am use the schema [action][reducer][subject] here.